code/game/machinery/bank_machine.dm

```diff
@@ -45,21 +46,22 @@
 		return
 	if (machine_stat & (BROKEN|NOPOWER))
 		say("Insufficient power. Halting siphon.")
-		end_syphon()
+		end_siphon()
 		return
 	var/siphon_am = 100 * delta_time
 	var/datum/bank_account/D = SSeconomy.get_dep_account(ACCOUNT_CAR)
 	if(!D.has_money(siphon_am))
 		say("Cargo budget depleted. Halting siphon.")
-		end_syphon()
+		end_siphon()
 		return
 
 	playsound(src, 'sound/items/poster_being_created.ogg', 100, TRUE)
 	syphoning_credits += siphon_am
 	D.adjust_money(-siphon_am)
 	if(next_warning < world.time && prob(15))
 		var/area/A = get_area(loc)
-		var/message = "Unauthorized credit withdrawal underway in [initial(A.name)]!!"
+		var/a = !unauthorized
```

Comment:
Use a better variable name. Additionally, it may just be worth just using the unauthorized variable, I don't see why you need another variable name for this

Reply:
localized it because else the line would be very long